### PR TITLE
[M1608-2] Benthic Photo Quadrat Transects Validations

### PR DESCRIPTION
**Pull request by gridcell**

### CODE REVIEW

**Comment by gridcell** [comment]
[Benthic Photo Quadrat Transect validations](https://trello.com/c/KNQQHTsu/1608-benthic-photo-quadrat-transect-validations)

**Comment by gridcell** [comment]
@saanobhaai Seem pieces having been commented out, specifically around viewsets and filters because they be implemented in another card

File: src/api/submission/parser.py [hunk 1]
```diff
@@ -229,3 +227,39 @@ def get_obs_colonies_bleached_data(collect_record, bleaching_quadrat_collection_
...
     return observations_data
+
+
+def get_quadrat_transect_data(collect_record, sample_event_id=None):
+    data = collect_record.data
+    quadrat_transect_data = data.get("quadrat_transect") or dict()
+    return dict(
+        sample_event=sample_event_id,
+        quadrat_size=_cast_decimal_to_str(quadrat_transect_data.get("quadrat_size")),
+        len_surveyed=quadrat_transect_data.get("len_surveyed"),
```
Comment (saanobhaai) [inline src/api/submission/parser.py:238]: Should this use `_cast_decimal_to_str` as well?

**Review by saanobhaai — CHANGES_REQUESTED** [review]
Just one question

**Pull request merged**